edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -42,12 +44,25 @@ public class Hosting
                             port,
                             listenOptions =>
                             {
-                                listenOptions.UseHttpsExtensions(
+                                listenOptions.UseHttps(
                                     new HttpsConnectionAdapterOptions()
                                     {
                                         ServerCertificate = serverCertificate,
-                                        ClientCertificateValidation = (clientCert, chain, policyErrors) => true,
-                                        ClientCertificateMode = certificateMode,
+                                        OnAuthenticate = (context, options) =>
+                                        {
+                                            if (clientCertAuthEnabled)
+                                            {
+                                                options.ClientCertificateRequired = true;
```

Comment:
>options.ClientCertificateRequired = true; [](start = 48, length = 41)

prob add a phat comment here explaining how DirectMethod works even though cert-required is true